server/lib/sync-agent.js

```diff
@@ -398,7 +400,7 @@ export default class SyncAgent {
     });
   }
 
-  startImportJob(url, partNumber, size) {
+  startImportJob(url, partNumber, size, import_id) {
```

Comment:
we could use the same approach as for `partNumber`, use camelCase as far as possible and then just set it to underscore_case in params